src/WordPress/PluginActions.php

```diff
@@ -81,6 +87,26 @@ public function patchPluginSettings()
         $path_array = explode('/', $this->request->getUrl());
         $settingId = $path_array[3];
 
+        $response = null;
+        if ($settingId === DataStore::DEFAULT_SETTINGS) {
+            $response = $this->patchPluginDefaultSettings();
+        } else {
+            $response = $this->patchPluginSettings2();
```

Comment:
Lets give this a better name, maybe rename this method `patchPluginSettingsRouter()` and then rename `patchPluginSettings2()` to `patchPluginSettings()`.